src/connection-manager/index.ts

```diff
@@ -525,6 +558,8 @@ export class DefaultConnectionManager extends EventEmitter<ConnectionManagerEven
         peerConnections.push(connection)
       }
 
+      connection.stat.direction = 'outbound'
```

Comment:
doesn't this gets assigned elsewhere?

Reply:
Not to my knowledge, which seemed to be [why this test fails otherwise](https://github.com/maschad/js-libp2p/blob/feat/seperate-limits-for-connections/test/connection-manager/index.spec.ts#L366)

I do see where a [stream gets assigned an outbound direction though](https://github.com/maschad/js-libp2p/blob/feat/seperate-limits-for-connections/src/connection/index.ts#L123) but that's after a stream is created

Reply:
@maschad That's stream direction, connection direction is set in connection upgrader: https://github.com/libp2p/js-libp2p/blob/master/src/upgrader.ts#L338

So it's very weird that you don't have direction in tests, maybe mock connection/upgrader isn't setting direction?

Reply:
Great point thank you, it turns out it was an issue with the mock upgrader. Fixed it now.